src/steps/devices/index.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  createDirectRelationship,
+  Entity,
+  IntegrationStep,
+  IntegrationStepExecutionContext,
+  RelationshipClass,
+} from '@jupiterone/integration-sdk-core';
+
+import { createAPIClient } from '../../client';
+import { IntegrationConfig } from '../../config';
+import {
+  ACCOUNT_ENTITY_KEY,
+  Entities,
+  Relationships,
+  Steps,
+} from '../constants';
+import { createDeviceEntity } from './converter';
+
+export async function fetchDevices({
+  instance,
+  jobState,
+}: IntegrationStepExecutionContext<IntegrationConfig>) {
+  const apiClient = createAPIClient(instance.config);
+
+  await apiClient.iterateDevices(async (device) => {
+    const deviceEntity = createDeviceEntity(device);
+
+    if (deviceEntity) {
+      await jobState.addEntity(deviceEntity);
+
+      await jobState.addRelationship(
+        createDirectRelationship({
+          _class: RelationshipClass.HAS,
+          from: (await jobState.getData(ACCOUNT_ENTITY_KEY)) as Entity,
```

Comment:
Similar comment here - please fetch the `accountEntity` above and then just do a `if` existence check and then create the relationships.